designer_v2/lib/repositories/api_client.dart

```diff
@@ -108,10 +123,15 @@ class StudyUApiClient extends SupabaseClientDependant with SupabaseQueryMixin im
    */
 
   @override
-  Future<List<Study>> getUserStudies({withParticipantActivity = true}) async {
+  Future<List<Study>> getUserStudies({withParticipantActivity = false, forDashboardDisplay = true}) async {
     await _testDelay();
     // TODO: fix Postgres policy for proper multi-tenancy
-    final columns = (withParticipantActivity) ? studyWithParticipantActivityColumns : studyColumns;
+    final columns = (withParticipantActivity)
+        ? studyWithParticipantActivityColumns
+        : (forDashboardDisplay)
+            ? studyDisplayColumns
+            : studyColumns;
+
```

Comment:
Could you add some `///` documentation to this function + explain the parameters